lib/rules/web/admin_console/4.2/overview.xyaml

```diff
@@ -126,3 +126,63 @@ group_by_label:
     dropdown_menu_item: <label>
   action: click_groupby_dropdown_button
   action: click_dropdown_menu_item
+click_workloads_tab:
+  element:
+    selector:
+      xpath: //a[contains(text(),'Workloads')]
+    op: click
```

Comment:
In lib/rules/web/admin_console/4.6/common_ui_elements.xyaml, there are many click tab rules, such as:
click_overview_tab:
  params:
    tab_name: Details
  action: click_tab
So click_workloads_tab also could use param, and put the rule in common_ui_elements.xyaml

Reply:
@yanpzhan moved the click_workloads_tab  to the common ui elements making use of click tab action.